### PR TITLE
feat: bigfloat library for rex calculations

### LINKED ISSUE #396: It's very possible this is causing a precision loss and therefore breaking rex calculations
https://github.com/telosnetwork/open-block-explorer/blob/f4d657f5683996140c189d67c0f7ab9c52826056/src/components/AccountCard.vue#L206

Using a value from mainnet for rexpool.total_rex and creating a number from it, we can see that we're starting to lose the last decimal of precision when it's an integer:

```
> m = new Number('947169457006.8053')
[Number: 947169457006.8053]
> m = new Number('9471694570068053')
[Number: 9471694570068052]
```

Should use a library that supports big decimal for the division to get rex price.

### PR DESCRIPTION
**Pull request by WikusKriek**
# Fixes #396 #420 

## Description

Fixed the precision loss by adding a big float package and using that to do price calculations.

## Checklist:

-   [x] I have performed a self-review of my own code
-   [x] I have commented my code, particularly in hard-to-understand areas
-   [x] I have cleaned up the code in the areas my change touches
-   [x] My changes generate no new warnings
-   [x] Any dependent changes have been merged and published in downstream modules
-   [x] I have checked my code and corrected any misspellings
-   [x] I have removed any unnecessary console messages

### CODE REVIEW

**Review by Viterbo — APPROVED** [review]

**Comment by donnyquixotic** [comment]
putting this on hold until refactor is merged

**Pull request closed without merging**
